src/resources/schema/document-toc.yml

```diff
@@ -28,7 +28,7 @@
 
 - name: toc-location
   schema:
-    enum: ["body", "left", "right"]
+    enum: ["body", "left", "right", "left-body", "right-body"]
   default: "right"
   tags:
     formats: [$html-doc]
```

Comment:
Could you also update the description below to include the new options? Might consider adding a short & long description and explaining these `-body` options a bit as they probably warrant a sentence to explain....